Packages/Tracking/Hands/Runtime/Scripts/HandModelManager.cs

```diff
@@ -0,0 +1,512 @@
+        /// <returns>Returns all hands which will be active if a hand appears in the scene</returns>
+        public List<HandModelBase> GetActiveHandsInScene()
+        {
+            List<HandModelBase> activeHands = new List<HandModelBase>();
+            foreach (HandModelPair handModelPair in handModelPairs)
+            {
+                if (!handModelPair.LeftEnableDisable.FreezeHandState ||
+                    handModelPair.LeftEnableDisable.FreezeHandState && handModelPair.Left.isActiveAndEnabled)
+                {
+                    activeHands.Add(handModelPair.Left);
+                }
+
+                if (!handModelPair.RightEnableDisable.FreezeHandState ||
+                    handModelPair.RightEnableDisable.FreezeHandState && handModelPair.Right.isActiveAndEnabled)
+                {
+                    activeHands.Add(handModelPair.Right);
+                }
+            }
+            return activeHands;
+        }
+
+        /// <summary>
+        /// Returns a hand model pair by index, if given a valid index
+        /// </summary>
+        /// <param name="index">The index of the hand model pair</param>
+        /// <param name="handModelPair">The hand model pair to be returned</param>
+        /// <returns>Returns true if the pair have been found, and false if not</returns>
+        public bool GetHandModelPair(int index, out HandModelPair handModelPair)
+        {
+            handModelPair = null;
+            if (index >= 0 && index < handModelPairs.Count)
+            {
+                handModelPair = handModelPairs[index];
+                return true;
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Returns a hand model pair by index, if given a valid index
+        /// </summary>
+        /// <param name="handModelPairID">The handModelPairID of the hand model pair</param>
+        /// <param name="handModelPair">The hand model pair to be returned</param>
+        /// <returns>Returns true if the pair have been found, and false if not</returns>
+        public bool GetHandModelPair(string handModelPairID, out HandModelPair handModelPair)
+        {
+            handModelPair = null;
+            if (handModelPairID != null)
+            {
+                handModelPair = handModelPairs.FirstOrDefault(hmp => hmp.HandModelPairId == handModelPairID);
+                return true;
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Returns a hand model by index, if given a valid index
+        /// </summary>
+        /// <param name="chirality">The chirality of the hand model</param>
+        /// <param name="index">The index of the hand model</param>
+        /// <param name="handModelBase">The hand model to be returned</param>
+        /// <returns>Returns true if the pair have been found, and false if not</returns>
+        public bool GetHandModel(Chirality chirality, int index, out HandModelBase handModelBase)
```

Comment:
Would be better named TryGetHandModel as it matches the signature of other .NET methods,

Reply:
✔️